common/models/user.js

```diff
@@ -312,6 +312,7 @@ module.exports = function(User) {
       ctx.where[pkName] = { inq: ids };
 
       AccessToken.destroyAll({ userId: { inq: ids }}, next);
+      console.log('tokens deleted for users %j', ids);
```

Comment:
> I just need it for testing. 

This is a code smell to me. The unit-tests should provide enough information when they fail so that no temporary console logs are needed.